### DISCUSSION TITLE
Please let me know how I can contribute to this project.

### DISCUSSION THREAD

**Discussion by ThinAaronGB** [community/community#38454] [Other Feature Feedback, Questions, & Ideas]
Hi! First, I'd like to say that your emulator is so cool! I've been using it since May of 2022 and I can't even tell you how impressive this emu is! The quality of the games in 4k plus the Vulkan system you operate in your emulator is out of this world! Thank you for all this really. I stopped using Duckstation for some time though, and I ended up deleting it altogether, then around September, I decided I wanted to reinstall it for the second time so off I went, however, this time some titles that I love, one of them being Silent Hill 1, began to crash while playing. It automatically took me out of the game with no apparent reason, something that had never happened back then in May, (first time I tried Duckstation on my MacBook Air M1) plus some other issues like in some video clips suddenly everything freezes out. I honestly would like to continue to use this emulator as in my opinion it's the best PS1 emulator under the sun and I love the design and the aesthetics of the games thanks to Duckstation and thanks to all of you who have made this possible. I hope you can work this/these issue(s) out. I don't know why this happens but I am 100 percent sure it did not happen before. Especially with this one game Silent Hill among some other titles too. Thank you for your time! Please let me know how I can contribute to this project so that I can help you somehow.
Kindest regards! Aaron.

**Comment by radar-blip** [comment]
This is a discussion forum. I think where you need to ask your question, and get in touch with the people who can answer you, 
are here: https://github.com/stenzek/duckstation

**Reply by ThinAaronGB** [reply]
Thanks for answering! The link you attached sends me back to the initial Duckstation main page. However, I'd like to contact someone within the project and I already tried contacting someone, but there's no way to go.

**Reply by radar-blip** [reply]
Contact owner of repo,  email is on profile.

**Comment by alimttth** [comment]
Per the discussion and suggestion [here](https://github.community/t/show-first-last-name-instead-of-account-name/2714/20), it seems like a lot of workplaces have developers use their personal github accounts in their organization, but there is no way to have github display the "N…

[Read more](https://github.com/orgs/community/discussions/10840)

**Reply by ThinAaronGB** [reply]
I'm sorry, I don't understand. What do you mean? Thanks for answering!
